sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_connection_async.py

```diff
@@ -349,12 +349,9 @@ async def _incoming_open(self, channel, frame):
             _LOGGER.info("<- %r", OpenFrame(*frame), extra=self._network_trace_params)
         if channel != 0:
             _LOGGER.error("OPEN frame received on a channel that is not 0.")
-            await self.close(
-                error=AMQPError(
-                    condition=ErrorCondition.NotAllowed,
-                    description="OPEN frame received on a channel that is not 0."
-                )
-            )
+            await self.close(error=AMQPConnectionError(
```

Comment:
This must be an AMQPError object - the ConnectionError is an Exception.... maybe we should consider renaming these...